src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -1057,5 +1057,75 @@ public override string VisualizeLocalType(object type)
         }
 
         #endregion
+
+        #region Span
+        //PROTOTYPE(span): this will be updated when rules for defining span are implemented
+        //                 most likely we would just pick the actual binary/corlib where
+        //                 span lives.
+        protected static CSharpCompilation CreateCompilationWithMscorlibAndSpan(string text, CSharpCompilationOptions options = null, CSharpParseOptions parseOptions = null)
+        {
+            var reference = CreateCompilation(
```

Comment:
Perhaps by making this an instance method and caching in an instance variable.

Reply:
There is a prototype comment below to remove this whole test utility when we reference the actual Span<>

---
In reply to: [133264014](https://github.com/dotnet/roslyn/pull/21006#discussion_r133264014) [](ancestors = 133264014)